src/main/java/de/tum/in/www1/artemis/service/dto/athena/ProgrammingFeedbackDTO.java

```diff
@@ -2,11 +2,14 @@
 
 import jakarta.validation.constraints.NotNull;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import de.tum.in.www1.artemis.domain.Feedback;
 
 /**
  * A DTO representing a Feedback on a ProgrammingExercise, for transferring data to Athena and receiving suggestions from Athena
  */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public record ProgrammingFeedbackDTO(long id, long exerciseId, long submissionId, String title, String description, double credits, Long structuredGradingInstructionId,
         String filePath, Integer lineStart, Integer lineEnd) implements FeedbackDTO {
```

Comment:
Similarly, the AthenaFeedbackSendingServiceTest is failing here.

We could set

```
        programmingFeedback.setText("some text");
        programmingFeedback.setDetailText("some detail text");
```

in the setup method or adjust the checks.